benchcab/internal.py

```diff
@@ -82,6 +82,88 @@
 # CABLE fixed C02 concentration
 CABLE_FIXED_CO2_CONC = 400.0
 
+# Contains the site ids for each met forcing file associated with an experiment
+# on modelevaluation.org
+MEORG_EXPERIMENTS = {
+    # List of site ids associated with the 'Five site test'
+    # experiment (workspace: NRI Land testing), see:
+    # https://modelevaluation.org/experiment/display/xNZx2hSvn4PMKAa9R
+    "five-site-test": [
+        "AU-Tum",
+        "AU-How",
+        "FI-Hyy",
+        "US-Var",
+        "US-Whs",
+    ],
+    # List of site ids associated with the 'Forty two site test'
+    # experiment (workspace: NRI Land testing), see:
+    # https://modelevaluation.org/experiment/display/urTKSXEsojdvEPwdR
+    "forty-two-site-test": [
+        "AU-Tum",
+        "AU-How",
+        "AU-Cum",
+        "AU-ASM",
+        "AU-GWW",
+        "AU-Ctr",
+        "AU-Stp",
+        "BR-Sa3",
+        "CA-Qfo",
+        "CH-Dav",
+        "CN-Cha",
+        "CN-Din",
+        "DE-Geb",
+        "DE-Gri",
+        "DE-Hai",
+        "DE-Tha",
+        "DK-Sor",
+        "FI-Hyy",
+        "FR-Gri",
+        "FR-Pue",
+        "GF-Guy",
+        "IT-Lav",
+        "IT-MBo",
+        "IT-Noe",
+        "NL-Loo",
+        "RU-Fyo",
+        "US-Blo",
+        "US-GLE",
+        "US-Ha1",
+        "US-Me2",
+        "US-MMS",
+        "US-Myb",
+        "US-NR1",
+        "US-PFa",
+        "US-FPe",
+        "US-SRM",
+        "US-SRG",
+        "US-Ton",
+        "US-UMB",
+        "US-Var",
+        "US-Whs",
+        "US-Wkg",
+    ]
+}
```

Comment:
A better idea to avoid distinctions between 1 site vs. other cases in the code:

```
for site_id in MEORG_EXPERIMENTS["five-site-test"]:
    MEORG_EXPERIMENTS[site_id] = [ site_id ]
```
That is adding each individual site id as an experiment id with a single value

Reply:
I think we should try to avoid writing for loops in a global namespace (see this [stackoverflow post](https://stackoverflow.com/questions/3611760/scoping-in-python-for-loops)), for example:
```python
for site_id in MEORG_EXPERIMENTS["five-site-test"]:
    MEORG_EXPERIMENTS[site_id] = [ site_id ]
```
the above for loop will make `site_id` accessible in the global scope.

I personally don't mind if the code is a bit more verbose since it makes it clear that specifying a single site must be a subset of the five sites. It also avoids duplication of data in `MEORG_EXPERIMENTS`